domains/restingbeast.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "RestingBeast",
+        "email": "mmmike2002@gmail.com"
+    },
+    "record": {
+        "CNAME": "restingbeast.github.io"
```

Comment:
I cannot reach your website (404 page)

Reply:
Sorry, it's still in development. I will reopen the PR when it's ready.